src/components/AutoUpdateTime.js

```diff
@@ -0,0 +1,86 @@
+import {View} from 'react-native';
+import React, {PureComponent} from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import styles from '../styles/styles';
+import DateUtils from '../libs/DateUtils';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import Timers from '../libs/Timers';
+import Text from './Text';
+
+const propTypes = {
+    timezone: PropTypes.shape({
+    // Value of selected timezone
+        selected: PropTypes.string,
+
+        // Whether timezone is automatically set
+        automatic: PropTypes.bool,
+    }),
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    timezone: null,
+};
+
+class AutoUpdateTime extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.getCurrentUserLocalTime = this.getCurrentUserLocalTime.bind(this);
+        this.state = {
+            timezone: this.getCurrentUserLocalTime(),
+        };
+    }
+
+    componentDidMount() {
+        this.timer = Timers.register(setInterval(() => {
+            this.setState({
+                timezone: this.getCurrentUserLocalTime(),
+            });
+        }, 60000));
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.timer);
+    }
+
+    getCurrentUserLocalTime() {
+        // if timezone isn't set, use local time instead.
```

Comment:
Capitalize first letter of comments
```suggestion
        // If timezone isn't set, use local time instead.
```